src/PrestaShopBundle/Resources/views/Admin/Layout/login_layout.html.twig

```diff
@@ -99,15 +112,14 @@
           <span class="login-back-shop">{{ shopName }}</span>
         </a>
 
-        {{ renderhook('displayAdminLogin') }}
```

Comment:
I'm not sure about this, I think this hook should be kept, right @jolelievre ?

Reply:
The hook has been re-added ✅
Sorry, I think I removed it during a rebase.